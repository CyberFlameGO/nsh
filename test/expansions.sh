a=$(( 1 ))
b=$(( 1 + 1 ))
c=$(( b + 1 ))
echo $a $b $c

x=$(echo $a)
y=$(echo $b)
echo $x
echo $y
echo $((x+y))

func1() {
    echo $(($1-1))
}
z=$(func1 7)
echo $((z*z))

words1="Cargo.toml Cargo.lock README.md"
ls $words1

IFS="/#"
words2="Cargo.toml/Cargo.lock#README.md"
ls $words2
echo "$words2"
IFS=" \t"
