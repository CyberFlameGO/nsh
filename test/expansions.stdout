1 2 3
1
2
3
36
Cargo.lock
Cargo.toml
README.md
Cargo.lock
Cargo.toml
README.md
Cargo.toml/Cargo.lock#README.md
